core/src/main/java/org/wso2/testgrid/infrastructure/InfrastructureCombinationsProvider.java

```diff
@@ -131,4 +136,44 @@ private void addSubpropertiesAsInfrastructureParameters(InfrastructureCombinatio
         }
     }
 
+    /**
+     * This class provides set of filtered infrastructure values.
+     * <p>
+     * The filtering is executed by reading exclude and include attributes in the @{@link TestgridYaml} file.
+     * If exclude attribute is set in @{@link TestgridYaml} and if those are exists in the current infrastructure
+     * value set, those will be eliminated.
+     * Further, if include attribute is set in @{@link TestgridYaml} and if those are exists in the current
```

Comment:
Same as the above comment.

Reply:
Fixed with e6479361dd8952f6b1efed92fdf164956691cf0e